scripts/query-endpoint-resources.sh

```diff
@@ -1,7 +1,10 @@
 #!/bin/sh
 
 #Iterate through endpoint source list json to query each url and store as properly named file
-cd ../resources/prod_resources
```

Comment:
I think we should update the main README so that it gives a more accurate description of what `make update_source_data` does, but that's my only comment!